### PR TITLE
Minor cleanup for for-equation listings

### PR DESCRIPTION
**Pull request by henrikt-ma**
Fixes whitespace issues and matches style of listing above.

### CODE REVIEW

File: chapters/equations.tex [hunk 1]
```diff
@@ -124,8 +124,8 @@ \subsubsection{Explicit Iteration Ranges of For-Equations}\label{explicit-iterat
   constant Integer j = 4;
   Real x[j]
 equation
-  for j in 1:j loop  // The loop-variable j takes the values 1,2,3,4
-    x[j] = j;        // Uses the loop-variable j
+  for j in 1 : j loop  // j takes the values 1, 2, 3, 4
```
Comment (HansOlsson) [inline chapters/equations.tex:127]:
```suggestion
  for j in 1:j loop    // j takes the values 1, 2, 3, 4
```
I don't think the white-space around ":" actually helps readability, and many libraries avoid it.

File: chapters/equations.tex [hunk 2]
```diff
@@ -138,10 +138,11 @@ \subsubsection{Implicit Iteration Ranges of For-Equations}\label{implicit-iterat
 
 \begin{example}
 \begin{lstlisting}[language=modelica]
-  Real x[n],y[n];
-for i loop          // Same as:  for i in 1:size(x ,1) loop
-  x[i] = 2*y[i];
-end for;
+  Real x[n], y[n];
+equation
+  for i loop          // Same as: for i in 1 : size(x, 1) loop
```
Comment (HansOlsson) [inline chapters/equations.tex:143]:
```suggestion
  for i loop          // Same as: for i in 1:size(x, 1) loop
```
Similarly.

File: chapters/equations.tex (hunk 1 above)
Comment (henrikt-ma) [reply chapters/equations.tex:127]: Applying suggestion just so that we can move on with this.

**Review by HansOlsson — APPROVED** [review]
Ok

**Pull request merged**